backend/users/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
at least one simple test should be required for when we eventually add in CI. but maybe right not it's not needed theres really only basic functionality.